Exercise/src/my_string.cpp

```diff
@@ -0,0 +1,67 @@
+#include "my_string.h"
+
+Exercise_1::CMyString::CMyString(const char* charsSequence) 
```

Comment:
please change the prefix according to documentation

Reply:
changed to csz prefix